### PR TITLE
rasterize lines and polygons

### PR DESCRIPTION
**Pull request by michaeldorman**

### CODE REVIEW

**Review by Nowosad — APPROVED** [review]

File: 06-raster-vector.qmd [hunk 1]
```diff
@@ -472,9 +473,87 @@ axes[1][0].set_title('Count')
...
-...
+Another dataset based on California's polygons and borders (created below) illustrates rasterization of lines. There are three preliminary steps. First, we subset the California polygon:
 
-## Spatial vectorization
+```{python}
+california = us_states[us_states['NAME'] == "California"]
+california
+```
+
+Second, we "cast" the polygon into a `"MultiLineString"` geometry, using the `.boundary` property that `shapely` geometries have:
```
Comment (Robinlovelace) [inline 06-raster-vector.qmd:483]: Great description.

File: 06-raster-vector.qmd [hunk 2]
```diff
@@ -472,9 +473,87 @@ axes[1][0].set_title('Count')
...
-## Spatial vectorization
+```{python}
+california = us_states[us_states['NAME'] == "California"]
+california
+```
+
+Second, we "cast" the polygon into a `"MultiLineString"` geometry, using the `.boundary` property that `shapely` geometries have: 
+
+```{python}
+california_borders = california['geometry'].iloc[0].boundary
```
Comment (Robinlovelace) [inline 06-raster-vector.qmd:486]: Interesting syntax. Why `.iloc[0]` is the question that comes to mind, would have assumed `california['geometry'].boundary` would work.

**Review by Robinlovelace — APPROVED** [review]
This looks great Michael.

File: 06-raster-vector.qmd (hunk 2 above)
Comment (michaeldorman) [reply 06-raster-vector.qmd:486]: > Interesting syntax. Why `.iloc[0]` is the question that comes to mind, would have assumed `california['geometry'].boundary` would work.

These are two methods to do the same thing, one using `shapely` and the other using `geopandas`. I agree it's better to be consistent, now changed to `geopandas` same as in the preceding example
https://github.com/geocompx/geocompy/commit/d81c253a341d73ee8dd83baaa400ac7578170074

**Pull request merged**